scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -315,6 +321,20 @@ public void initActivity(Body body) {
             logger.debug("Booting jmx...");
             this.jmxHelper.boot(authentication);
 
+            privateStore = PAActiveObject.newActive(AOSynchronization.class,
+                                                    new Object[] { PASchedulerProperties.getAbsolutePath(PASchedulerProperties.SCHEDULER_SYNCHRONIZATION_DATABASE.getValueAsString()) });
```

Comment:
In the end, why do you need to instantiate a `privateStore` typed with `AOSynchronization`? I don't see any method specific to `AOSynchronization` being called.

Reply:
I understand that this part might be confusing. I will add comments to clarify. I really need to create an AOSynchronization active object.
newActive(SomeInterface.class) makes no sense (though turnActive would). 
I prefer to rely on newActive here (matter of taste).

Then, I also need a "public" stub object, a stub which does not drag to the node all the dependencies that the concrete implementation does.